tests/test_market_aggregator.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import os
+from time import sleep
+from d3a_api_client.aggregator import Aggregator
+from d3a_api_client.utils import get_area_uuid_from_area_name_and_collaboration_id
+from d3a_api_client.rest_market import RestMarketClient
+
+
+class TestAggregator(Aggregator):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.is_finished = False
+        self.fee_cents_kwh = 0
```

Comment:
```suggestion
        self.fee_cents_per_kwh = 0
```
minor, just to be clear